ruth/simulator/singlenode.py

```diff
@@ -75,18 +77,15 @@ def simulate(
 
             with timer_set.get("alternatives"):
                 need_new_route = [vehicle for vehicle in vehicles_to_be_moved if
-                                  vehicle.is_at_the_end_of_segment(self.sim.routing_map)]
-                alts = alternatives_provider.compute_alternatives(
-                    need_new_route,
-                    k=self.sim.setting.k_alternatives
-                )
-                alts = remove_infinity_alternatives(alts, self.sim.routing_map)
-                assert len(alts) == len(need_new_route)
+                                  vehicle.is_at_the_end_of_segment(self.sim.routing_map)
+                                  and vehicle.behavior != VehicleBehavior.DEFAULT]
```

Comment:
It seems like `VehicleBehavior.DEFAULT` means that the car won't ever re-schedule its path (where does it get its original path)? It would be nice to document these behaviours, e.g. on the `VehicleBehavior` enum.